scripts/_common.sh

```diff
@@ -33,6 +33,18 @@ config_nginx() {
 }
 
 config_monitorix() {
+    jail_list=$(fail2ban-client status | grep 'Jail list:' | sed 's/.*Jail list://' | sed 's/,//g')
+    additional_jail=""
+    for jail in $jail_list; do
+        if ! [[ "$jail" =~ (recidive|pam-generic|yunohost|postfix|postfix-sasl|dovecot|nginx-http-auth|sshd|sshd-ddos) ]]; then
```

Comment:
Can this test be done also for the "default" rules ?
Just to be sure that they exist

Reply:
Don't really understand what you mean

Reply:
Can we be sure that everyone have jails named recidive|pam-generic|yunohost|postfix|postfix-sasl|dovecot|nginx-http-auth|sshd|sshd-ddos ?
If one is not active, there will be massive logs about that.

Reply:
Normally they should exist as it should be provided by default on Yunohost. I'll make a deeper check.